src/Surfnet/Stepup/Tests/Configuration/Event/EventSerializationAndDeserializationTest.php

```diff
@@ -47,8 +47,11 @@
 use Surfnet\Stepup\Configuration\Value\NumberOfTokensPerIdentityOption;
 use Surfnet\Stepup\Configuration\Value\RaLocationId;
 use Surfnet\Stepup\Configuration\Value\RaLocationName;
+use Surfnet\Stepup\Configuration\Value\SelectRaaOption;
 use Surfnet\Stepup\Configuration\Value\ShowRaaContactInformationOption;
+use Surfnet\Stepup\Configuration\Value\UseRaaOption;
 use Surfnet\Stepup\Configuration\Value\UseRaLocationsOption;
+use Surfnet\Stepup\Configuration\Value\UseRaOption;
 use Surfnet\Stepup\Configuration\Value\VerifyEmailOption;
 use Surfnet\StepupBundle\Value\SecondFactorType;
 
```

Comment:
Do the `select`- and `use`- events not need to be tested?

Reply:
In the context of serialization, they are tested in the very test case you are commenting on. Or are you suggesting another type of test? I'm all ears!

Reply:
I ment I did mis the `Value`-tests for the `Options`. Sorry for mentioning `events`.

Reply:
Ah, yes, adding a test for value objects representing the FGA options might be a good idea. And I've considered this. Given the very low complexity of said objects. I've restrained from doing this. But now I'm starting to think otherwise.